code/Model/Submission/SubmittedFileField.php

```diff
@@ -87,53 +109,67 @@ public function getFormattedValue()
      */
     public function getExportValue()
     {
-        return ($link = $this->getLink()) ? $link : '';
+        return ($links = $this->getLinks()) ? implode("\r", $links) : '';
     }
 
     /**
      * Return the link for the file attached to this submitted form field.
      *
-     * @return string
+     * @return array|null
      */
-    public function getLink($grant = true)
+    public function getLinks($grant = true)
```

Comment:
We can't just rename methods outside of a major release.
Either target this to the `7` branch or implement a new method and deprecate the old one.